misk/src/test/kotlin/misk/web/actions/NotFoundActionTest.kt

```diff
@@ -110,14 +169,35 @@ class NotFoundActionTest {
     return Request.Builder()
         .post(content.toRequestBody(contentType))
         .url(jettyService.httpServerUrl.newBuilder().encodedPath(path).build())
-        .header("Accept", acceptedMediaType.toString())
+        .apply {
+          if (acceptedMediaType != null)
+            header("Accept", acceptedMediaType.toString())
```

Comment:
Does the request blow up if Accept header isn't passed in?

Reply:
Heh, yeah it sends the string "null" as the accept header value, which throws
```
java.lang.IllegalArgumentException: null is not a valid media range
	at misk.web.mediatype.MediaRange$Companion.parse(MediaRange.kt:87)
	at misk.web.mediatype.MediaRange$Companion.parseRanges(MediaRange.kt:81)
	at misk.web.HttpCall$DefaultImpls.accepts(HttpCall.kt:101)
	at misk.web.ServletHttpCall.accepts(ServletHttpCall.kt:13)
	at misk.web.jetty.WebActionsServlet.handleCall(WebActionsServlet.kt:65)
	at misk.web.jetty.WebActionsServlet.doPost(WebActionsServlet.kt:51)
...
```